sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -618,10 +604,11 @@ internal virtual Response<ConfigurationSetting> GetConfigurationSetting(string k
         }
 
         /// <summary>
-        /// Retrieves one or more <see cref="ConfigurationSetting"/> that satisfies the options of the <see cref="SettingSelector"/>
+        /// Retrieves one or more <see cref="ConfigurationSetting"/> that satisfy the options set in the <see cref="SettingSelector"/>.
         /// </summary>
-        /// <param name="selector">Set of options for selecting <see cref="ConfigurationSetting"/> from the configuration store.</param>
+        /// <param name="selector">Options used to select a set of <see cref="ConfigurationSetting"/>s from the configuration store.</param>
```

Comment:
Tip: I used to add an "s" on the end of type/member references, but it often comes through looking weird, like:

> `ConfigurationSetting`s.

What I've been doing instead is something like:

> `ConfigurationSettings` values